README.md

```diff
@@ -26,40 +26,49 @@ You can try the [annotation demo](http://doccano.herokuapp.com).
 
 ## Usage
 
-Two options to run doccano:
+Two options to run doccano: production or development.
```

Comment:
The list is preferable:

- Production
- Development

Reply:
Done.